azure-pipelines/createBuildDirectories.sh

```diff
@@ -58,6 +58,7 @@ requiredLocDirectories=(
     "pl"
     "pl-PL"
     "pt-br"
+    "pt-BR"
```

Comment:
Can you remove the `pt-br`? If not, do you know why this only affects some of the langs (for example, `ja-JP` doesn't seem to be affected by this?)

Reply:
Finally figured out what's causing this issue. This problem happens with pt-BR, zh-Hant and zh-Hans.

- Net 472 stores these langs as        pt-BR, zh-Hant and zh-Hans 
- While net 7 stores these langs as   pt-br, zh-hant and zh-hans

While this is not a problem in windows as dirs are case insensitive, this causes a problem in linux as some projects which only target net7.0 won't have net472 required dirs created in their bin folder when we build those projects. This causes unsuccessful copy attempts and eventually fails the build. 

It does not happen in case of ja-JP as both net 472 and net 7 uses ja folder to store the loc files.

Reply:
Thanks for the investigation. Can you add this as a comment here for future reference?